src/main/java/Nehsik.java

```diff
@@ -0,0 +1,93 @@
+import java.util.Scanner;
+public class Nehsik {
+    public static final int MAX_TASKS = 100;
+
+    public static void main(String[] args) {
```

Comment:
Avoid Long methods. The main function can be SLAP/refactored into separate methods to improve readability and also make the code easier to extend and maintain.